contracts/TokenVault.sol

```diff
@@ -178,6 +189,35 @@ contract TokenVault is Ownable, Recoverable {
     return token.balanceOf(address(this));
   }
 
+  /// @dev Check how many tokens "investor" can claim
+  /// @param investor Address of the investor
+  /// @return uint How many tokens the investor can claim now
+  function getClaimableAmount(address investor) public constant returns (uint claimableAmount) {
```

Comment:
since this returns the amount of tokens that the given address can claim *now*, it should return 0 until now > freezeEndsAt, no matter the branches, as the description says.